src/features/import/hooks/useConfigure.ts

```diff
@@ -36,44 +63,46 @@ export default function useConfigure(orgId: number) {
       dispatch(
         importPreviewAdd({
           problems: [],
-          stats: {
-            person: {
-              summary: {
-                addedToOrg: {
-                  byOrg: {},
-                  total: 0,
-                },
-                created: {
-                  total: 0,
-                },
-                tagged: {
-                  byTag: {},
-                  total: 0,
-                },
-                updated: {
-                  byChangedField: {},
-                  byInitializedField: {},
-                  total: 0,
-                },
-              },
-            },
-          },
+          stats: emptyStats,
         })
       );
     } else {
-      const importOperations = prepareImportOperations(
-        configuredSheet,
-        countryCode
-      );
-
-      const previewRes = await apiClient.post<
-        ImportPreview,
-        ZetkinPersonImportPostBody
-      >(`/api/orgs/${orgId}/bulk/preview`, {
-        ops: importOperations,
-      });
+      let importOperations;
+      try {
+        importOperations = prepareImportOperations(
+          configuredSheet,
+          countryCode
+        );
+      } catch (error) {
+        addUnexpectedErrorToProblems();
```

Comment:
Don't we have to return here? If we catch this error and then proceed, `importOptions` should be undefined but on line 81 we will still try to submit it to the server. I think we need to stop here by returning.

Reply:
I agree! I will make the change!